tests/foreman/cli/test_docker.py

```diff
@@ -254,7 +255,7 @@ def test_update_docker_repo_upstream_name(self):
 
     @tier1
     @run_only_on('sat')
-    def test_update_docker_repo_url(self):
+    def test_positive_update_docker_repo_url(self):
```

Comment:
`test_positive_update_url`
You probably got the idea, please update the rest of tests accordingly.